conda_gitenv/deploy.py

```diff
@@ -54,6 +54,11 @@ def deploy_tag(repo, tag_name, target):
 
 
 def create_env(pkgs, target, pkg_cache):
+    # Empty package cache so that it will reinitialised.
+    conda.install.package_cache_ = {}
```

Comment:
>isn't there a conda api function to control the package cache

not that I know of or can find?
The following is with a checked out copy of conda 4.1.12:
```
$ grep -ir package_cache
fetch.py:from .install import (add_cached_package, find_new_location, package_cache, dist2pair,
fetch.py:    for dist, info in iteritems(package_cache()):
install.py:    res = package_cache().get(dist, {}).get('urls', (None,))
install.py:package_cache_ = {}
install.py:    package_cache()
install.py:    rec = package_cache_.get(fkey)
install.py:        rec = package_cache_[fkey] = dict(files=[], dirs=[], urls=[])
install.py:def package_cache():
install.py:    if package_cache_:
install.py:        return package_cache_
install.py:    package_cache_['@'] = None
install.py:    del package_cache_['@']
install.py:    return package_cache_
install.py:    package_cache()
install.py:    rec = package_cache().get(dist)
install.py:    return set(dist for dist, rec in package_cache().items() if rec['files'])
install.py:    for fn in package_cache().get(dist, {}).get('files', ()):
install.py:    rec = package_cache().get(dist)
install.py:    del package_cache_[dist]
install.py:    return set(dist for dist, rec in package_cache().items() if rec['dirs'])
install.py:    for fn in package_cache().get(dist, {}).get('dirs', ()):
install.py:    rec = package_cache().get(dist)
install.py:        del package_cache_[dist]
install.py:    rec = package_cache()[dist]
```